### PR TITLE
Feature/events snake grid

### PR DESCRIPTION
**Pull request by noway**
Snakey Wakey: the events page redesign

### CODE REVIEW

File: src/components/EventCard.vue [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+  data() {
+    return {
+    }
+  },
+
+  computed: {
+    width() {
+      const { cellSize, gap } = this;
+      const { is_highlighted } = this.item.event_listing;
```
Comment (JacobKorn) [inline src/components/EventCard.vue:43]: Generally I would convert `is_highlighted` to `isHighlighted` at some boundary, just to fit with js conventions.

File: src/components/EventCardMobile.vue [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+  </a>
+</template>
+
+<script>
+import { getSrcSet, addHost } from './responsiveImage'
+
+export default {
+  name: 'event-card-mobile',
+
+  components: {
```
Comment (JacobKorn) [inline src/components/EventCardMobile.vue:65]: empty components

File: src/components/EventCardMobile.vue [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+    }
+  },
+  methods: {
+    addHost(imageSrc) {
+      return addHost(this.host, imageSrc);
+    },
+  },
+  data() {
+    return {
+      baseUrl: process.env.BASE_URL,
```
Comment (JacobKorn) [inline src/components/EventCardMobile.vue:89]: is this used anywhere?

File: src/components/EventCardThreeByTwo.vue [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+  },
+
+  methods: {
+    addHost(imageSrc) {
+      return addHost(this.host, imageSrc);
+    },
+  },
+  data() {
+    return {
+      baseUrl: process.env.BASE_URL,
```
Comment (JacobKorn) [inline src/components/EventCardThreeByTwo.vue:91]: this doesn't seem to be used either.

File: src/components/EventsSnakeGrid.vue [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+    </div>
+  </div>
+</template>
+
+<script>
+import { getAreas, generateGridHybrid, clearCanvasSnake, drawCanvasSnake } from './snakeGrid'
+
+export default {
+  name: "events-snake-grid",
+  components: {
```
Comment (JacobKorn) [inline src/components/EventsSnakeGrid.vue:42]: empty components

File: src/components/EventsSnakeGrid.vue [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+    gap: {
+      type: Number,
+      default: 10,
+    },
+    lineWidth: {
+      type: Number,
+      default: 50,
+    },
+  },
+  data() {
```
Comment (JacobKorn) [inline src/components/EventsSnakeGrid.vue:113]: empty data

File: src/views/EventsSnakeGridScaffold.vue [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+  },
+  data() {
+    return {
+      items: defaultItems,
+      baseUrl: process.env.BASE_URL,
+      isShown: true,
+    }
+  },
+
+  computed: {
```
Comment (JacobKorn) [inline src/views/EventsSnakeGridScaffold.vue:148]: empty computed

**Comment by noway** [comment]
@JacobKorn all fixed, latest build pushed up here also: https://github.com/TEDxAuckland/tedx-auckland-site/pull/113 

All good to merge from my side.

**Comment by JacobKorn** [comment]
Good work man. Thank you.

**Pull request merged**